README.md

```diff
@@ -40,6 +40,18 @@ Thank you for contributing! If you're unfamiliar with the code, start by reading
 
 Please fork this repository and contribute back using [pull requests](https://github.com/thunderbird/thunderbird-android/pulls).
 
+We use git submodules for some dependencies. To clone the repository, use the following command:
```

Comment:
In the past we were using submodules and some (drive-by) contributors struggled quite a bit because of that. After a while we got rid of them again. I've heard similar stories from other open source projects. That's why I think we should avoid using submodules for anything affecting typical community contributions.

I think using a submodule for the metadata is a good idea. And having that in a submodule shouldn't affect people who want to make code changes. Knowing about (or having to clone) the metadata submodule isn't necessary to make changes to the code and building the app.

We could either change this first sentence to be more specific, e.g. "We use a git submodule for the app metadata used by F-Droid", or we could move this information from the README to some other location (because it's not really necessary for casual contributors to know about it), maybe `RELEASING.md`.